ml_utils/ml_utils_cli/cli/apps/projects.py

```diff
@@ -245,7 +251,10 @@ def add_prediction(
             threshold = 0.1
 
         model = YOLO(model_name)
-        model.set_classes(labels)
+        if hasattr(model, "set_classes"):
```

Comment:
this was from my previous PR, but it is running and error when I have a locally pre-trained model, hence this handling method